stores/sql/mysql/migrations/main/migration_00014_hosts_resolvedaddresses.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE hosts DROP COLUMN subnets;
```

Comment:
I think you haven't updated `schema.sql` yet for both mysql and sqlite right? So the code seems to still be using the `resolved_addresses` column that the migrations are getting rid of.

Reply:
Sorry, I think did a rebase but there were some conflicts so I missed the commit that changed the schema when going back manually.